docusaurus/docs/reactnative/common-content/contexts/channels-context/loading_next_page.mdx

```diff
@@ -0,0 +1,5 @@
+True if `ChannelList` component is loading more channels as part of pagination. This will be set to true, when user scrolls to bottom of the list and [`hasNextPage`](#hasnextpage) is true.
```

Comment:
this one is just an opinion: i feel like the comma on `set to true, ` its not needed